pkg/api/controller.go

```diff
@@ -49,6 +50,18 @@ func (c *Controller) Run() error {
 		os.Exit(1)
 	}
 
+	// Updating the CVE Database
+	if c.Config != nil && c.Config.Extensions != nil && c.Config.Extensions.Search != nil &&
+		c.Config.Extensions.Search.CVE.UpdateInterval > 1 {
+		go func() {
+			err := cveinfo.UpdateCVEDb(c.Config.Storage.RootDirectory, c.Log,
+				c.Config.Extensions.Search.CVE.UpdateInterval, false)
```

Comment:
What happens when queries arrive when this update is in progress? Is something serializing it?

Reply:
Queries has to wait when they are trying to read from boltdb.